src/sidebar/store/modules/defaults.js

```diff
@@ -0,0 +1,59 @@
+import * as util from '../util';
+
+/**
+ * A store module pertaining to client-side defaults for (browser) users.
+ */
+
+function init() {
+  /**
+   * Note that the persisted presence of any of these defaults cannot be
+   * guaranteed, so consumers of said defaults should be prepared to handle
+   * missing (i.e. `null`) values. As `null` is a sentinal value indicating
+   * "not set/unavailable", a `null` value for a default is otherwise invalid.
+   */
+  return {
```

Comment:
I have a mild concern that, because the store module here is so agnostic about where its values come from, it may seem a little "magic" to future devs that the `persistedDefaults` service is behind the scenes here, manipulating things. Does that seem like a risk for confusion?

Reply:
Hmm, maybe. You could add a comment here that the `persistedDefaults` service handles persistence of these value.

The store's responsibility in contrast is to provide the model for the UI to render and services to react to.